### DISCUSSION TITLE
My language not showing

### DISCUSSION THREAD

**Discussion by leonardocastro-dev** [github-linguist/linguist#6337] [Classification] [answered]
my language is not showing in my repository, link to repository - https://github.com/leonardodecastro-programmer/curso_ebac_frontend/tree/exerc%C3%ADcio_css

**Comment by lildude** [comment, marked as answer]
This is because Linguist only analyses the default branch and your default branch [`exercicio_git`](https://github.com/leonardodecastro-programmer/curso_ebac_frontend/tree/exercicio_git) only has a single text file.

**Reply by leonardocastro-dev** [reply]
thank you bro
